test/C/src/federated/FederatedFileInclude.lf

```diff
@@ -0,0 +1,13 @@
+/** Success here is just successfully compiling. */
```

Comment:
We already have this test?

Reply:
I already put a very similar test under `target` because the `files` target property is only supported by the C target currently.